# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../apmd/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual fixes to make the code leagal C for gcc-3.4.

  - Rene Rebe <rene@exactcode.de>

--- ./apmd.c.vanilla	2004-05-26 17:53:37.000000000 +0200
+++ ./apmd.c	2004-05-26 17:56:55.000000000 +0200
@@ -71,8 +71,8 @@
 
 
 #ifdef APM_TRACE
-#define ADEBUG(lev,args...) \
-    if (APM_TRACE>=lev) syslog(LOG_DEBUG, __FUNCTION__ ": " args)
+#define ADEBUG(lev, text, args...) \
+    if (APM_TRACE>=lev) syslog(LOG_DEBUG, "%s : " text, __FUNCTION__, args)
 #else
 #define ADEBUG(args...)
 #endif
